tests/PurgeDatabaseTrait.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Tests;
+
+use Doctrine\Common\DataFixtures\Purger\ORMPurger;
+use Doctrine\ORM\EntityManagerInterface;
+use PHPUnit\Framework\Attributes\Before;
+use Symfony\Bundle\FrameworkBundle\Test\KernelTestCase;
+
+trait PurgeDatabaseTrait
+{
+    #[Before]
+    protected function _purgeDatabase(): void
```

Comment:
The same CS question.

```suggestion
    protected function purgeDatabase(): void
```

Reply:
I've totally removed this trait in next PR.